src/view_element.cpp

```diff
@@ -1586,7 +1586,7 @@ void View::DrawRestWhole(DeviceContext *dc, int x, int y, int valeur, bool cueSi
     if (y > (int)staff->GetDrawingY()
         || y < staff->GetDrawingY()
                 - (staff->m_drawingLines - 1) * m_doc->GetDrawingDoubleUnit(staff->m_drawingStaffSize))
-        DrawHorizontalLine(dc, x1, x2, y1, m_doc->GetDrawingStaffLineWidth(staff->m_drawingStaffSize));
+        DrawHorizontalLine(dc, x1, x2, y1, m_doc->GetDrawingStaffLineWidth(staff->m_drawingStaffSize) * 1.75);
```

Comment:
1.75 would be good to prepare for styling.  Maybe a class to store style settings?  This variable could be accessed with:

`Style::GetLedgerLineToStaffLineRatio()`

And could be changed with:

`Style::SetLedgerLineToStaffLineRatio()`

and/or a function called:

`m_doc->GetDrawingLedgerLineWidth(staff->m_drawingStaffSize()`

could be added.